WORKSPACE.bazel

```diff
@@ -0,0 +1,7 @@
+workspace(name = "json-schema-validator")
+
+load("//:workspace.bzl", "load_nlohmann_3_11_2")
+load("//:workspace.bzl", "load_nlohmann_3_11_3")
```

Comment:
Does this mean that 2 versions of `nlohmann/json` are included for the user, or that it supports only 2 versions? Are there other designs that can specify version ranges that are supported?

Reply:
The goal was to specify which version of `nlohmann/json` the project should use. Imagine a situation when `json-schema-validator` is linked into another project, which already uses `nlohmann/json` library. I wanted to use the same version of `nlohmann/json`.  
There is a default version, but a user could overwrite `nlohmann/json` version. I just tried versions 3_11_2 and 3_11_3 and they passed unit tests, but it does not mean that they are the only supported versions.